website/settings/defaults.py

```diff
@@ -306,6 +306,9 @@ def parent_dir(path):
         },
     }
 
+WATERBUTLER_JWE_SALT = 'yusaltydough'
```

Comment:
Should these settings be added to set of required settings in `settings/__init__.py`?